apps/schools/domains/common/constants/Table.ts

```diff
@@ -0,0 +1,2 @@
+export const searchColumns = ['phone', 'position', 'name']
```

Comment:
it is strange
will we write there all names of columns? I thouth we can define it in special compnents